recipes/hankel/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "hankel" %}
+{% set version = "0.3.9" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 6c2c0ba5143efd6556e657349ca7927e0db3821a23e3458a3748f71c792de2a5
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
+    - scipy >=0.12.0
+    - mpmath >=0.18
+    - numpy >=1.6.1
+    - future  # [py2k]
```

Comment:
The future package is also available for python 3. So you can remove the `[py2k]` selector to make the package fully `noarch` compatible.

Reply:
I am okay with that solution.